Spigot-Server-Patches/0616-Allow-individual-plugins-to-load-datapacks.patch

```diff
@@ -0,0 +1,201 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Thonk <30448663+ExcessiveAmountsOfZombies@users.noreply.github.com>
+Date: Tue, 15 Dec 2020 09:07:01 -0600
+Subject: [PATCH] Allow individual plugins to load datapacks
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/datapack/PluginDataPack.java b/src/main/java/com/destroystokyo/paper/datapack/PluginDataPack.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..ec8d5c32ea14bc4941a318d9b8177627950ebfa3
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/datapack/PluginDataPack.java
+@@ -0,0 +1,59 @@
++package com.destroystokyo.paper.datapack;
```

Comment:
Paper is moving to the io.papermc package

Reply:
Thanks!